common/remote/sockfile.cpp

```diff
@@ -5889,12 +5906,31 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
 #endif
                         }
                     }
-                    catch (IException *e)
+                    catch (IJSOCK_Exception *e)
                     {
+                        // accept failed ...
                         EXCLOG(e,"CRemoteFileServer (secure)");
                         e->Release();
                         break;
                     }
+                    catch (IException *e) // IDAFS_Exception also ...
+                    {
+                        EXCLOG(e,"CRemoteFileServer1 (secure)");
+                        e->Release();
+                        if (sockSSL)
+                        {
+                            sockSSL->shutdown();
+                            sockSSL->close();
+                            sockSSL.clear();
+                        }
+                        if (ssock)
+                        {
+                            ssock->shutdown();
+                            ssock->close();
+                            ssock.clear();
+                        }
+                        securesockavail = false;
+                    }
                 }
```

Comment:
I am not sure - I think both shutdown() and close() sould throw a IJOCK_Exception that I should catch and handle.